pyproject.toml

```diff
@@ -36,6 +36,11 @@ Source = "https://github.com/zytedata/duplicate-url-discarder"
 [project.optional-dependencies]
 rules = ["duplicate-url-discarder-rules"]
 
+[tool.pytest.ini_options]
+filterwarnings = [
+    'ignore::DeprecationWarning:scrapy',
+]
```

Comment:
Why is this needed?

Reply:
An unrelated warning I noticed while running the tests:

```
tests/test_pipelines.py::test_duplicate_url_discarder_pipeline_no_addon
  /home/adrian/temporal/duplicate-url-discarder/.tox/py/lib/python3.13/site-packages/scrapy/core/downloader/webclient.py:10: DeprecationWarning: twisted.web.http.HTTPClient was deprecated in Twisted 24.7.0: Use twisted.web.client.Agent instead.
    from twisted.web.http import HTTPClient
```

Reply:
Looks like a deprecation error affecting Scrapy itself in the download handler of `data:` URLs.

Reply:
Right, so it's not a `ScrapyDeprecationWarning` filter.

Reply:
(the related issue is https://github.com/scrapy/scrapy/issues/6450)